File: CodeSnippets/c codesnippets/18July/prog3.c

#include<stdio.h>
#define Core2web(a) a

void main() {

        int a = 1;

        switch(a) {

                case Core2web(1):
                        printf("PPA\n");
                        break;

                case Core2web(2):
                        printf("JAva\n");
                        break;

        }

}
/*
vivek@vivek-HIRAY:~/Desktop/CodeSnippets/c codesnippets/18July$ cc prog3.c
vivek@vivek-HIRAY:~/Desktop/CodeSnippets/c codesnippets/18July$ ./a.out 
PPA
 */

File: CodeSnippets/javaSnippets/5Aug/prog3.java


class Core2web {

	public static void leftshift(int var1, int var2) {

		var1 <<= var2;

	}
	public static void main(String[] args) {

		int var1 = 4, var2 = 2;
		leftshift(var1, var2);
		System.out.println(var1);

	}

}
/*
vivek@vivek-HIRAY:~/Core2Web-TREE-/CodeSnippets/javaSnippets/5Aug$ javac prog3.java 
vivek@vivek-HIRAY:~/Core2Web-TREE-/CodeSnippets/javaSnippets/5Aug$ java Core2web 
4
 */
